datalad/distribution/siblings.py

```diff
@@ -239,30 +246,47 @@ def __call__(
         # at the top-level and vice versa
         worker = action_worker_map[action]
 
-        dataset = require_dataset(
-            dataset, check_installed=False, purpose='configure sibling')
-        refds_path = dataset.path
+        ds = require_dataset(
+            dataset,
+            # it makes no sense to use this command without a dataset
+            check_installed=True,
```

Comment:
FWIW, it was `=False` in the original PR https://github.com/datalad/datalad/pull/1500/files#diff-a45159f0547073e97cc9b38ee5d4fb94ecf3b6ec6d70250e5017d074d976deccR184 so not like it was set to `=False` for some explained in some later commit/PR reason, and I agree with the comment.